cmake/modules/falcosecurity-libs.cmake

```diff
@@ -50,6 +50,8 @@ if(MUSL_OPTIMIZED_BUILD)
 endif()
 
 set(SCAP_HOST_ROOT_ENV_VAR_NAME "HOST_ROOT")
+set(SCAP_HOSTNAME_ENV_VAR "FALCO_HOSTNAME")
```

Comment:
There is an existing `FALCO_HOSTNAME` env variable which could be replaced by the new scap agent info we added to scap (not in this PR obviously).

The other one is completely new and open to any naming conventions as as you said it will become a contract.

If you also referred to the `_NAME` suffix yes we can add that.